### PR TITLE
fix(application-shell-connectors): prevent Sentry reports when `additionalClaims` value is falsy

### PR DESCRIPTION
**Pull request by kark**
Fixes https://jira.commercetools.com/browse/SHIELD-616

### CODE REVIEW

File: packages/application-shell-connectors/src/components/application-context/application-context.tsx [hunk 1]
```diff
@@ -90,7 +90,7 @@ export const mapUserToApplicationContextUser = (user?: TFetchedUser) => {
     let additionalClaims: Record<string, unknown> = {};
     try {
       additionalClaims = JSON.parse(
-        user.idTokenUserInfo.additionalClaims || ''
+        user.idTokenUserInfo.additionalClaims || '{}'
```
Comment (kark) [inline packages/application-shell-connectors/src/components/application-context/application-context.tsx:93]: I’m not sure, should empty all falsy values fall back to parsable JSON here?

File: test-data/user/types.ts [hunk 2]
```diff
@@ -30,7 +30,7 @@ type TBaseUser = {
     iat: number;
     email?: string | null | undefined;
     name?: string | null | undefined;
-    additionalClaims?: string;
+    additionalClaims?: string | null;
```
Comment (kark) [inline test-data/user/types.ts:33]: Taken from real-life scenario
Screenshot 2022-10-14 at 15 32 42

**Review by CarlosCortizasCT — APPROVED** [review]
Looks good to me.

Thanks 👍

File: packages/application-shell-connectors/src/components/application-context/application-context.tsx (hunk 1 above)
Comment (tdeekens) [reply packages/application-shell-connectors/src/components/application-context/application-context.tsx:93]: To me that's okay as it eases consumption a bit as you don't always have to guard against the additionalClaims being nil.

**Review by tdeekens — APPROVED** [review]
Danke :)

**Review by ddouglasz — APPROVED** [review]
Looks good 👏

**Review by emmenko — APPROVED** [review]
💯

**Pull request merged**